Scripts/Connector/Sample connector restore script.ps1

```diff
@@ -0,0 +1,44 @@
+# This script restores a connector that has been backed up.
+# To backup a Connector look at "Sample connector backup script.ps1"
+
+# BEFORE RUNNING THESE COMMANDS PERFORM THE FOLLOWING STEPS:
+# 1) Copy the C:\ConnectorBackup folder (generated by the backup script) to the machine in which you will restore it
+# 2) Install the latest Connector from https://aka.ms/UPConnector (do no register the Connector)
+ 
+# RUN THIS SCRIPT IN AN ELEVATED POWERSHELL WINDOW
+#Requires -RunAsAdministrator
+
+# Stop services
+net stop "Print Connector service"
+net stop PrintConnectorUpdaterSvc
+ 
+# Restore local printers
+c:\windows\system32\spool\tools\PrintBrm.exe -R -F C:\ConnectorBackup\printers.brm
+ 
+# Import UP certificates
+New-Item -Path "CERT:\LocalMachine\PrintProxyStore"
+$pwd = ConvertTo-SecureString -String "password" -AsPlainText -Force
+Get-ChildItem "C:\ConnectorBackup\Certs" | Foreach-Object { Import-PfxCertificate -Password $pwd -FilePath $_.FullName -CertStoreLocation Cert:\localmachine\PrintProxyStore -Exportable }
+ 
+# Import connector registries
+reg import C:\ConnectorBackup\Connector.reg
+ 
+# Import the Cloud Data registries
+Get-ChildItem "C:\ConnectorBackup\CloudData" | Foreach-Object { reg import $_.FullName }
+ 
+# Copy Connector config json
+copy C:\ConnectorBackup\config.json C:\windows\PrintConnectorSvc\
+ 
+# Copy Custom Mappings
+xcopy /S /Y c:\ConnectorBackup\PrintTicketMappings\ C:\ProgramData\Microsoft\UniversalPrintConnector\CustomPrintTicketMappings
+
+# Restore permissions for certificates
+icacls "C:\ProgramData\Microsoft\Crypto\RSA" /restore c:\ConnectorBackup\certificatePermissions.txt
+
+Write-Host "ENSURE THE CONNECTOR SERVICE HAS BEEN STOPPED IN THE OLD MACHINE BEFORE PROCEEDING." -ForegroundColor Yellow
+Write-Host "Press ENTER to start the restored Connector."
+Read-Host
+
+# Start services
+net start "Print Connector service"
+net start PrintConnectorUpdaterSvc
```

Comment:
Either as a warning (or even doing it for them) in the script or in the documentation, we need them to delete the backup because it contains certs. Both on the source and destination machines.

Reply:
I don't think we should do it ourselves (they may want to save it to do a future restore if needed)
I think it can go with the rest of the notes in the documentation:
1) Change the drive if needed
2) Copy over the backup folder
3) Delete the backup folder if no longer needed